c2corg_api/models/utils.py

```diff
@@ -13,7 +13,10 @@ def copy_attributes(obj_from, obj_to, attributes):
     Copies the given attributes from `obj_from` to `obj_to` (shallow copy).
     """
     for attribute in attributes:
-        if hasattr(obj_from, attribute):
+        # has_geom_attribute isnt a real column in a table
+        # it is only defined in the models
+        # we avoid looking for it here making travis break
```

Comment:
Do the tests pass locally, but not on Travis?

Reply:
sorry by travis I mean all the tests. 
They broke locally and on travis before

Reply:
`copy_attributes` is called in `DocumentGeometry.to_archive/update`:
https://github.com/c2corg/v6_api/blob/8844ae9dc72faa083e608cd8ecadbc46f859ab88/c2corg_api/models/document.py#L340-L346

Any reason `has_geom_detail` must be listed in `DocumentGeometry._ATTRIBUTES`?

Reply:
I don't see any reason indeed, i will removed it.